optaplanner-docs/src/main/asciidoc/MoveAndNeighborhoodSelection/MoveAndNeighborhoodSelection-chapter.adoc

```diff
@@ -331,15 +329,78 @@ Advanced configuration:
     </pillarSwapMoveSelector>
 ----
 
+For a description of `subPillarType` and related properties, please refer to <<subPillars,Subpillars>>.
```

Comment:
we generally don't put that S in a capital, unless of course it's something we always capticalize (such as Tabu Search).